### PR TITLE
* [android] fix loading NPE when use dom object

### PR DESCRIPTION
**Pull request by sospartan**
Fix NPE in `WXLoading` when component is destroyed.

### CODE REVIEW

File: android/sdk/src/main/java/com/taobao/weex/ui/component/WXLoading.java [hunk 1]
```diff
@@ -241,14 +242,16 @@ protected WXFrameLayout initComponentHostView(@NonNull Context context) {
 
   @Override
   public void onLoading() {
-    if (getDomObject().getEvents().contains(Constants.Event.ONLOADING)) {
+    ImmutableDomObject domObject = getDomObject();
+    if (domObject != null && domObject.getEvents().contains(Constants.Event.ONLOADING)) {
```
Comment (YorkShen) [inline android/sdk/src/main/java/com/taobao/weex/ui/component/WXLoading.java]: 像下面的那一个修改一样，判断domObject.getEvents()是否为空

    domObject != null &&
    domObject.getEvents() !=null &&
    domObject.getEvents().contains(Constants.Event.ONLOADING)

Comment (sospartan) [inline android/sdk/src/main/java/com/taobao/weex/ui/component/WXLoading.java]: `getEvents`是NonNull的

**Comment by sospartan** [comment]
rebase了一下 diff不太对  我重发一个

**Comment by cxfeng1** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><strong><a href="https://github.com/alibaba/weex/blob/f5913220b0e71a97bde80a8a981f3deda2e76e3d/ios/sdk/WeexSDK/Sources/Utility/WXConvert.h#L14">ios/sdk/WeexSDK/Sources/Utility/WXConvert.h#L14</a></strong>: include of non-modular header inside framework module ‘WeexSDK.WXConvert’ [-Werror,-Wnon-modular-include-in-framework-module]  <br /><code>
#import "WXBoxShadow.h"
</code></td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><strong><a href="https://github.com/alibaba/weex/blob/f5913220b0e71a97bde80a8a981f3deda2e76e3d/ios/sdk/WeexSDKTests/WXSDKEngineTests.m#L10">ios/sdk/WeexSDKTests/WXSDKEngineTests.m#L10</a></strong>: could not build module ‘WeexSDK’  <br /><code>
#import "WXSDKEngine.h"
</code></td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR may need tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Pull request closed without merging**